### PR TITLE
[ADD] Install dependencies from 10, saas15 and saas17

### PR DESCRIPTION
**Pull request by ruiztulio**

### CODE REVIEW

File: odoo-shippable/scripts/build-image.sh [hunk 1]
```diff
@@ -57,6 +57,12 @@ PIP_DEPENDS_EXTRA="line-profiler watchdog coveralls diff-highlight \
                    pg-activity virtualenv nodeenv setuptools==33.1.1 \
                    html2text==2016.9.19 ofxparse==0.15"
 PIP_DPKG_BUILD_DEPENDS=""
+
+ODOO_DEPENDENCIES="git+https://github.com/vauxoo/odoo@10.0 \
+                   git+https://github.com/vauxoo/odoo@saas-15 \
```
Comment (JesusZapata) [inline odoo-shippable/scripts/build-image.sh]: @ruiztulio 
The branch `saas-14` is not neccessary ?

Comment (ruiztulio) [inline odoo-shippable/scripts/build-image.sh]: @JesusZapata they (saas-14 ans saas-15) have the same requirements, but not saas-17

**Comment by JesusZapata** [comment]
@moylop260 
What do you think about this change ?
Now on the `saas-17` has the follow trouble 

```
Testing tanner:
copy /home/odoo/data_dir/filestore/openerp_template /home/odoo/data_dir/filestore/openerp_test
coverage run /home/odoo/odoo-saas-17/odoo-bin -d openerp_test --stop-after-init --log-level info --log-handler openerp.tools.yaml_import:DEBUG --log-handler openerp.models.schema:DEBUG --init tanner
2017-08-11 19:31:52,122 197 INFO ? odoo: Odoo version 10.saas~17
2017-08-11 19:31:52,122 197 INFO ? odoo: Using configuration file at /home/odoo/.openerp_serverrc
2017-08-11 19:31:52,123 197 INFO ? odoo: addons paths: [u'/home/odoo/data_dir/addons/10.saas~17', u'/home/odoo/build/Vauxoo/tanner', u'/home/odoo/server-tools', u'/home/odoo/enterprise', u'/home/odoo/addons-vauxoo', u'/home/odoo/odoo-saas-17/addons', u'/home/odoo/odoo-saas-17/odoo/addons', '/.repo_requirements/odoo/odoo/addons']
2017-08-11 19:31:52,123 197 INFO ? odoo: database: default@default:default
2017-08-11 19:31:52,156 197 INFO openerp_test odoo.modules.loading: loading 1 modules...
2017-08-11 19:31:52,169 197 CRITICAL openerp_test odoo.modules.module: Couldn't load module base
2017-08-11 19:31:52,169 197 CRITICAL openerp_test odoo.modules.module: No module named PyPDF2
2017-08-11 19:31:52,169 197 ERROR openerp_test odoo.modules.registry: Failed to load registry
Traceback (most recent call last):
File "/.repo_requirements/odoo/odoo/modules/registry.py", line 84, in new
odoo.modules.load_modules(registry._db, force_demo, status, update_module)
File "/.repo_requirements/odoo/odoo/modules/loading.py", line 281, in load_modules
loaded_modules, processed_modules = load_module_graph(cr, graph, status, perform_checks=update_module, report=report)
File "/.repo_requirements/odoo/odoo/modules/loading.py", line 122, in load_module_graph
load_openerp_module(package.name)
File "/.repo_requirements/odoo/odoo/modules/module.py", line 368, in load_openerp_module
__import__('odoo.addons.' + module_name)
File "/.repo_requirements/odoo/odoo/modules/module.py", line 82, in load_module
exec(open(modfile, 'rb').read(), new_mod.__dict__)
File "<string>", line 4, in <module>
File "/home/odoo/odoo-saas-17/odoo/addons/base/ir/__init__.py", line 9, in <module>
from . import ir_actions_report
File "/home/odoo/odoo-saas-17/odoo/addons/base/ir/ir_actions_report.py", line 22, in <module>
from PyPDF2 import PdfFileWriter, PdfFileReader
ImportError: No module named PyPDF2
2017-08-11 19:31:52,170 197 CRITICAL openerp_test odoo.service.server: Failed to initialize database `openerp_test`.
```

The dependency `PyPDF2` is present on [saas-17/requirements.txt](https://github.com/Vauxoo/odoo/blob/saas-17/requirements.txt#L25)

This change fix that trouble

**Comment by moylop260** [comment]
@JesusZapata 
It error shouldn't reproduced since that MQT install odoo/requirements.txt so you need review why is not installing it?

**Pull request merged**